Mathlib/SetTheory/Game/Relabelling.lean

```diff
@@ -0,0 +1,334 @@
+/-
+Copyright (c) 2019 Kim Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kim Morrison
+-/
+import Mathlib.SetTheory.Game.Basic
+
+/-!
+# Relabelling
+
+`PGame.Relabelling` is more of a technical detail of computable constructions in Lean than
```

Comment:
I think this docstring should reflect our actual discussions: this file is deprecated, and any computational functionality should be done through `Short` instead.